package.json

```diff
@@ -1,7 +1,18 @@
 {
-  "name": "vue-camptocamp",
-  "version": "0.1.0",
-  "private": true,
+  "name": "c2c_ui",
+  "description": "https://www.camptocamp.org user interface",
+  "version": "7.0.0beta",
```

Comment:
Do we really need to start from 7.0?

Reply:
We start from whatever we want. Increasing major version seems to be obvious, but that's a number that may cause some reaction. I've hesitate to skip 7, and go directly to 8 for this reason :) 

Whatever, what is your opinion on this ?